todo-app/src/todoState.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+class todoState extends React.Component {
```

Comment:
Component names should be named using [pascal case](http://wiki.c2.com/?PascalCase)!